src/Servers/testassets/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <!-- Tests do not work on Helix or when bin/ directory is not in project directory due to undeclared dependency on test content. -->
+    <!-- Local testing does not work when bin/ directory is not in project directory due to deployer expectations. -->
```

Comment:
nit: I don't think this was about "test content". Problem seems rooted in the various deployers.